README.md

```diff
@@ -25,7 +25,7 @@ v2.x - CQ 5.6
 
 Active development is on the "master" branch, which is currently the 4.x version line. Security patches and the like are sometimes back-ported to prior versions.
 
-Of course pull-requests are happily accepted for those that would like to submit things like support for AEM 6.0, back-porting features for AEM 5.6, etc.
```

Comment:
Because up above we say:

'''
v4.x - AEM 6.1
v3.x - CQ 5.6 and AEM 6.0
v2.x - CQ 5.6
'''